[BUG] dead reference in getting-started's sample code

**Describe the bug**
In a sample code of getting-started (typescript), there is dead file reference (typo?).

In https://github.com/awslabs/cdk8s/blob/master/docs/getting-started/typescript.md#constructs , sample code of `WebService` definition:

```
import { Construct, Node } from 'constructs';
import { Deployment, Service, IntOrString } from './imports/k8s';

export interface WebServiceOptions {
...
```

The doc says,

> To implement WebService, create a file lib/web-service.ts (the convention is to use lib for reusable components):

and if the code is `./lib/web-service.ts`'s content instead of `./web-service.ts`, `import { ... } from './imports/k8s';` cannot be resolved.
`import { ... } from '../imports/k8s';` is ok.

**To Reproduce**
Proceed along getting-started/typescript docs from a head, and make `lib/web-service.ts` with the sample code as is.
Typescript compiler claim `Cannot find module './imports/k8s'`.

**Expected behavior**
In sample code,
```
import { Deployment, Service, IntOrString } from '../imports/k8s';
```

**Additional context**

--- a/docs/getting-started/typescript.md
+++ b/docs/getting-started/typescript.md
@@ -246,7 +246,7 @@ to use `lib` for reusable components):
 
 ```ts
 import { Construct, Node } from 'constructs';
-import { Deployment, Service, IntOrString } from './imports/k8s';
+import { Deployment, Service, IntOrString } from '../imports/k8s';
 
 export interface WebServiceOptions {
   /**

